opal/core/api.py

```diff
@@ -300,6 +301,7 @@ def list(self, request):
             [e.to_dict(request.user) for e in Episode.objects.all()]
         )
 
+    @transaction.atomic
```

Comment:
If we're doing this for these... do we not want them for everything? Why just episodes ? Do we really want the whole function wrapped in it? I think we only really want e.g.  from create patient to episode update_from_dict ?